code/obj/item/implant.dm

```diff
@@ -1655,7 +1655,7 @@ ABSTRACT_TYPE(/obj/item/implant/revenge)
 				. += "It appears to contain \a [src.imp.name] with [I.uses] charges."
 
 	unlimited
-		implant_type = "/obj/item/implant/access/infinite"
+		implant_type = /obj/item/implant/access/infinite
```

Comment:
i don't see any text2path removals for `implant_type`?

Reply:
![image](https://user-images.githubusercontent.com/53602251/189368758-2e11d593-0b53-4553-bed6-bdbbe751b7fc.png)
this is how implants did it
![image](https://user-images.githubusercontent.com/53602251/189368829-e632657f-5a20-42ab-b4ee-5882724051d3.png)
(actual implant code here)